### PR TITLE
Update readme, add instructions for payment permit locally

### PR DESCRIPTION
**Pull request by web4er**
This PR doesn't resolve an existing issue. 
This PR does the following:
- update outdated instructions in the readme file
- add a manual method to setup payment permits and private keys related to it for a local instance of ubiquibot

Why didn't I add an automatic method? 
It appears the automatic method https://pay.ubq.fi/onboarding is for clients who are using an official instance of ubiquibot. Because it doesn't provide x25519_PRIVATE_KEY for the encrypted private key which is needed in a local instance.

QA-part1: https://github.com/web4er/ubiquibot/issues/79
QA-part2: https://github.com/web4er/ubiquibot/issues/78

### CODE REVIEW

**Review by rndquu — APPROVED** [review]

**Comment by rndquu** [comment]
> It appears the automatic method https://pay.ubq.fi/onboarding is for clients who are using an official instance of ubiquibot. Because it doesn't provide x25519_PRIVATE_KEY for the encrypted private key which is needed in a local instance.

Exactly

File: README.md
Comment (whilefoo) [inline README.md:174]: can you just change to capital letter X because env names are case-sensitive

**Review by whilefoo — CHANGES_REQUESTED** [review]
Look good but I fear some new contributors might be confused because there are 2 ways to run the bot, as github action or locally. 

In [How to QA any additions to the bot](https://github.com/ubiquity/ubiquibot/blob/75561cd99f9f578e3cb8e88b651e6fc6fb8236b9/README.md#how-to-qa-any-additions-to-the-bot) we instruct them to fork the repo, install qa bot and enable actions as if they will use it as a github action but in [Payments Permits in a local instance](https://github.com/ubiquity/ubiquibot/blob/75561cd99f9f578e3cb8e88b651e6fc6fb8236b9/README.md#payments-permits-in-a-local-instance) we instruct them to put `X25519_PRIVATE_KEY` in the `.env`, but if they are using github actions they need to put it in the workflow.

File: README.md
Comment (pavlovcik) [inline README.md:55]: Do we still have `/ 10`?

File: README.md
Comment (pavlovcik) [inline README.md:79]:
```suggestion
`payment-permit-max-price` sets the max amount for automatic payout of bounties when the issue is closed
```

File: README.md
Comment (pavlovcik) [inline README.md:81]:
```suggestion
`comment-incentives` can be `true` or `false` that enable or disable comment incentives. These are payments generated for comments in the issue by contributors, excluding the assignee.
```

File: README.md
Comment (pavlovcik) [inline README.md:85]: Will add 0.1

Not sure why my GitHub client isn't letting me suggest changes anymore

File: README.md
Comment (pavlovcik) [inline README.md:83]: "Incentive reward" Is redundant

File: README.md
Comment (pavlovcik) [inline README.md:87]:
```suggestion
`max-concurrent-assigns` is the maximum number of bounties that can be assigned to a bounty hunter at once. This excludes bounties with delayed or approved pull request reviews.
```

File: README.md
Comment (pavlovcik) [inline README.md:89]:
```suggestion
`register-wallet-with-verification` can be `true` or `false`. If enabled, it requires a signed message to set wallet address. This prevents users from setting wallet address from centralized exchanges, which would make payments impossible to claim.
```

File: README.md
Comment (pavlovcik) [inline README.md:91]:
```suggestion
`promotion-comment` is a message that is appended to the payment permit comment.
```

**Review by pavlovcik — CHANGES_REQUESTED** [review]

File: README.md
Comment (web4er) [reply README.md:174]: > Look good but I fear some new contributors might be confused because there are 2 ways to run the bot, as github action or locally.

I can fix this by adding following paragraph after the heading.


**You can perform a QA using your local instance of ubiquibot, or ubiquibot-qa app. The following steps describe how you can perform a QA using ubiquibot-qa.**


But do they need to change `X25519_PRIVATE_KEY` when they are using the QA bot? Encrypted private key is read from org config. And they don't have access to it, so they will not need to know this workflow key. What do you think?

Comment (whilefoo) [reply README.md:174]: `X25519_PRIVATE_KEY` for github actions is here https://github.com/ubiquity/ubiquibot/blob/d56a523c81829e17513f7da1f1ccb1bbdca8b180/.github/workflows/bot.yml#L81

The problem I see with QAing with github actions is that they will want to change the workflow variables to use their own instance of supabase and logger...which will need a commit that will show up in the pull request so when they finish work they need to revert changes so we don't merge them to our development branch. 

I think we should only write instructions for running the bot locally because that's the easiest way to debug and QA, at least in my experience.

File: README.md
Comment (web4er) [reply README.md:55]: It appears it is 100 now.

Even though looks like 1000.
https://github.com/ubiquity/ubiquibot/blob/d56a523c81829e17513f7da1f1ccb1bbdca8b180/src/handlers/shared/pricing.ts#L7

but priority is being being divided by 10 too. 
https://github.com/ubiquity/ubiquibot/blob/d56a523c81829e17513f7da1f1ccb1bbdca8b180/src/handlers/shared/pricing.ts#L6

File: README.md
Comment (web4er) [reply README.md]: I like this. Fewer complications at starting stage. I have updated docs to QA using local instances. I am marking it resolved, but feel free to mark it unresolved if something needs a change.

**Comment by web4er** [comment]
I have made the required changes. This PR is ready for a new review. @pavlovcik @whilefoo

File: README.md
Comment (pavlovcik) [inline README.md:55]: Is / 100 still accurate?

**Review by pavlovcik — DISMISSED** [review]

File: README.md
Comment (whilefoo) [inline README.md:172]:
```suggestion
2. Go to repository settings and turn on `issues`.
```

Comment (web4er) [reply README.md:172]: I see these actions inform the hunter about the wrong commit message format and lint problems before they create PR preventing the need for a forced push later. Sounds like a part of QA to me.

File: README.md
Comment (web4er) [reply README.md:55]: oh that is multiply, not divide.

File: README.md
Comment (whilefoo) [reply README.md:172]: Hmm okay I was just worried the bot will be triggered by actions but if the QA bot is not installed on the repo it will just fail, so it's fine

File: README.md
Comment (web4er) [reply README.md]: I have tested this to make sure I am not wrong https://github.com/web4erOrg/ubiquibot/issues/58

**Comment by web4er** [comment]
Need to approve this if you think your concerns have been addressed. @whilefoo @pavlovcik

**Review by pavlovcik — APPROVED** [review]

**Review by whilefoo — APPROVED** [review]

**Comment by web4er** [comment]
@rndquu please approve this PR so that it can be merged.

**Review by rndquu — APPROVED** [review]

**Pull request merged**
